src/com/amazon/corretto/crypto/provider/HpkeGen.java

```diff
@@ -0,0 +1,49 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.amazon.corretto.crypto.provider;
+
+import java.security.*;
```

Comment:
Don't use wildcard imports. Ideally, just let the IDE import everything for you.

Reply:
Oops, this was IntelliJ overriding me, updated my settings to prevent it from doing this.